mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -191,10 +191,13 @@ class DistanceSensorPlugin : public MavRosPlugin {
 			return;
 		}
 
-		if (sensor->orientation >= 0 && dist_sen.orientation != sensor->orientation) {
```

Comment:
I'm not understanding this. `orientation` param is supposed to be a string (like `"LOCAL_NED"`). Can you suggest a way of receiving as a string a treat it as an integer? Cause that's what I'm trying to do since ever!